test-workspace/test.proto

```
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+package test;
+
+message TestMessage {
+    string name;
```

Buf (1.54.0)

field test.TestMessage.name: missing field tag number

(COMPILE)